packages/esm-ohri-core-app/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["src/**/*"],
+  "include": ["src/**/*", "../esm-commons-lib/src/components/patient-lists/patient-list.component.tsx", "../esm-commons-lib/src/components/patient-lists/patient-list.component.tsx"],
```

Comment:
Why do you need this? I think this config needs to go to the target package: https://github.com/UCSF-IGHS/openmrs-esm-ohri/blob/master/packages/esm-commons-lib/tsconfig.json

Reply:
Let me revert this